vision/snippets/crop_hints/crop_hints.py

```diff
@@ -24,17 +24,27 @@
 # [START vision_crop_hints_tutorial_imports]
 import argparse
 
+from typing import MutableSequence
+
 from google.cloud import vision
 from PIL import Image, ImageDraw
+
 # [END vision_crop_hints_tutorial_imports]
 
 
-def get_crop_hint(path):
+def get_crop_hint(path) -> MutableSequence[vision.Vertex]:
```

Comment:
Please also add type hint for the path argument.

Reply:
Added.